doc/lua_api.txt

```diff
@@ -1549,15 +1549,36 @@ since, by default, no schematic attributes are set.
 Items
 =====
 
+Items are things that can be held by players and stored in inventories.
+Items come in the form of item stacks, which is a collection of equal
+items that occupies a single inventory slot.
+
 Item types
 ----------
 
 There are three kinds of items: nodes, tools and craftitems.
 
-* Node: Can be placed in the world's voxel grid
-* Tool: Has a wear property but cannot be stacked. The default use action is to
-  dig nodes or hit objects according to its tool capabilities.
-* Craftitem: Cannot dig nodes or be placed
+* Node: Placable item form of a node in the world's voxel grid
```

Comment:
```suggestion
* Node: Place-able item form of a node in the world's voxel grid
```
This suggestion is _really_ arguable, but I figured I'd field it since 'placable' is an existing word that means something else and is pronounced with a hard c.